rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -342,6 +342,31 @@ namespace rocal{
             .def_readwrite("process_time", &TimingInfo::process_time)
             .def_readwrite("transfer_time", &TimingInfo::transfer_time);
         py::class_<rocalTensor>(m, "rocalTensor")
+            .def(
+                "__add__",
+                [](rocalTensor *output_tensor, rocalTensor *output_tensor1)
+                {
+                    py::object fn_module = py::module::import("amd.rocal.fn");
+                    auto fn_function_call = fn_module.attr("tensor_add_tensor_float")(output_tensor, output_tensor1).cast<RocalTensor>();
+                    return fn_function_call;
```

Comment:
Please rename the variable name fn_function_call to fn_call/function_call

Reply:
Changed